config/locales/gobierto_data/views/ca.yml

```
@@ -43,16 +43,20 @@ ca:
       guide: Guia per utilitzar l'editor SQL
       info: Informació
       modifiedQuery: Consulta modificada
+      modifiedVisualization: Visualització modificada
       news: Notícies
       private: Privat
       queries: Consultes
       queryExecuted: Consulta executada en
       queryName: Nom consulta
       recents: Recents
       records: Registres
+      resetQuery: Restableix la consulta
+      resetViz: Restableix la Visualització
```

Keep consistency between languages:
```suggestion
      resetQuery: Restablir consulta
      resetViz: Restablir Visualització
```